Adjust documentation for event system after recent API changes (#5224)
microsoft/MixedRealityToolkit-Unity: Documentation/Input/InputEvents.md

> [!IMPORTANT]
> By default a script will receive events only while in focus by a pointer. To receive events while out of focus, in addition to implementing the desired handler interfaces, you have to do one of the following:

> > - (Preferred) Register the script's game object as a global listener via [`MixedRealityToolkit.InputSystem.RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem) in `OnEnable` (`UnregisterHandler` explicitly in `OnDisable`). You need to call `RegisterHandler` multiple times, if you want to listen to global input events on multiple interfaces.

Reviewer: we're avoiding "You" in the docs

Reviewer: ```suggestion
> - (Preferred) Register the script's game object as a global listener via [`MixedRealityToolkit.InputSystem.RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem) in `OnEnable` (`UnregisterHandler` explicitly in `OnDisable`). `RegisterHandler` needs to be called multiple times, for listening to global input events on multiple interfaces.
```

> > - (Preferred) Register the script's game object as a global listener via [`MixedRealityToolkit.InputSystem.RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem) in `OnEnable` (`UnregisterHandler` explicitly in `OnDisable`). You need to call `RegisterHandler` multiple times, if you want to listen to global input events on multiple interfaces.

Reviewer 2: It'd be good to explain why this is preferred over the other one.

> > - Derive the script from [`InputSystemGlobalHandlerListener`](xref:Microsoft.MixedReality.Toolkit.Input.InputSystemGlobalHandlerListener). It will add an `inputSystem` property to the component and make sure inputSystem is valid before registering handlers. Implement its abstract methods `RegisterHandlers` and `UnregisterHandlers`. In these implementations use [`MixedRealityToolkit.InputSystem.RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem) to register interfaces, which you want to receive global events for.

Reviewer: ```suggestion
> - Derive the script from [`InputSystemGlobalHandlerListener`](xref:Microsoft.MixedReality.Toolkit.Input.InputSystemGlobalHandlerListener). This will add an `inputSystem` property to the component. Also make sure inputSystem is valid before registering handlers. Implement its abstract methods `RegisterHandlers` and `UnregisterHandlers`. In these implementations [`MixedRealityToolkit.InputSystem.RegisterHandler`](xref:Microsoft.MixedReality.Toolkit.IMixedRealityEventSystem) should be used to define the interfaces the global events should be received for.
```
